roles/ceph-docker-common/defaults/main.yml

```diff
@@ -2,3 +2,9 @@
 ceph_conf_overrides:
   global:
     log_file: /dev/null
+
+# The number of times to retry the docker image pull
+download_retry: 3
```

Comment:
please run `generate_group_vars_sample.sh` instead of writing `group_vars/docker-commons.yml.sample`

Reply:
I would prefer a variable name with a docker prefix. ``docker_download_retry`` or something like this. It is also ``docker_timeout`` and not ``timeout``.

Reply:
@berendt it makes sense since it's related to docker operations.
I would even say : `s/docker_timeout/docker_pull_timeout`
@jtaleric can you please update your PR with what has been said?

Reply:
@guits ``docker_pull_timeout`` makes sense, yes. It is not a generic timeout.